src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -44,6 +44,8 @@ case class ExecutionServiceLogs(
  logs: Map[String, Seq[Map[String,String]]]
 )
 
+case class ExecutionServiceWorkflowOptions(jes_gcs_root: String)
```

Comment:
This class is to encapsulate all the options so its name is appropriate. Future options should be placed in this case class and possibly jes_ces_root could become Option[String] but it seemed silly to do that now.